PLEP-0001.md

```diff
@@ -0,0 +1,176 @@
+
+If a topic is already covered by an existing PLEP, then it is often
+appropriate to propose an amendment to that PLEP instead of writing a
+new PLEP.  Amendments should not substantially change the spirit of
+the PLEP.  Proposed amendments should update the "date last revised"
+category in the header.  Amendments to PLEPs go through the same
+process as newly proposed PLEPs.
+
+When major changes to a PLEP are desired, then it is most appropriate
+to propose a new PLEP to supercede the old PLEP.  The pull request to
+supersede a PLEP should change the "status" category in the header of
+the old PLEP to "superseded by PLEP nn" where nn is the number of the
+new PLEP.
+
+When PLEPs are amended or superseded, it is generally helpful to
+request reviews from the authors of the original PLEP.
+
+Minor changes to a PLEP that do not affect its meaning or intent
+(e.g., formatting changes, fixes to typos, updates to links, minor
+rewording to improve clarity, and reversing cosmic ray bit flips) do
+not need to go through the full review process, but do require
+approval by the Coordinating Committee.
+
+### Submitting a PLEP
+
+All new PLEPs and amendments to PLEPs should be submitted as pull
+requests into the PlasmaPy-PLEP repository.  The pull request may be
+created while the PLEP is being written to allow for greater
+transparency and community input during the writing process.  In this
+case, the sponsor should comment on the pull request and inform the
+Coordinating Committee when the proposed PLEP is ready for review.
+
+### Review process
+
+When a new PLEP or an amendment to a PLEP is submitted, the
+Coordinating Committee should appoint one of its members to be the
+PLEP's editor.  The editor is responsible for aiding the sponsor by
+making sure that the PLEP follows the accepted standard and
+facilitating communication between the sponsor and the Coordinating
+Committee.  The editor and sponsor may be the same person.
+
+When the editor agrees that the PLEP is reasonably close to the
+accepted standard, then the sponsor, the editor, or another member of
+the Coordinating Committee should email the PlasmaPy list to introduce
+the PLEP and request community feedback.  The discussion should take
+place through normal communication channels.  Important points and
+relevant meeting minutes should be recorded as comments on the pull
+request so that the community discussion is archived as much as
+possible.  During the discussion phase, the sponsor should revise the
+PLEP in order to address concerns raised by the community.  The
+discussion should ideally continue until a general consensus among the
+PlasmaPy community has been reached.  
+
+The Coordinating Committee is tasked with making the final decision on
+the PLEP.  A PLEP shall be accepted when approved by at least
+two-thirds of the Coordinating Committee.
+
+### PLEP status
+
+The status of a standard or process PLEP may be any of the following:
```

Comment:
Can an informational PLEP have status or is it omitted?

Reply:
I was wondering about this.  The status for informational enhancement proposals for other projects is usually "Discussion" but this doesn't seem quite the best way to go about it.  Maybe omitting the status of informational PLEPs would be best.

Reply:
Maybe. But then explicit is better than implicit, right? Perhaps just 'information' would do?